net-misc/gns3-server/gns3-server-2.1.20.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{5,6} )
```

Comment:
What's blocking py3.7? Maybe we could test it.

Reply:
definitely fails, see aiohttp, python-zipstream and other dependencies that do not support 3.7